starters/adapters/aws-lambda/src/entry_aws-lambda.tsx

```diff
@@ -1,68 +1,22 @@
 /*
  * WHAT IS THIS FILE?
  *
- * It's the entry point for the Express HTTP server when building for production.
+ * It's the entry point for Aws Lambda when building for production.
  *
- * Learn more about Node.js server integrations here:
- * - https://qwik.builder.io/docs/deployments/node/
+ * Learn more about the Aws Lambda integration here:
+ * - https://qwik.builder.io/docs/deployments/aws/
  *
  */
-import "source-map-support/register";
 import {
   createQwikCity,
-  type PlatformNode,
-} from "@builder.io/qwik-city/middleware/node";
+  type PlatformAwsLambda,
+} from "@builder.io/qwik-city/middleware/aws-lambda";
 import qwikCityPlan from "@qwik-city-plan";
 import { manifest } from "@qwik-client-manifest";
-import serverless from "serverless-http";
```

Comment:
since you remove this, I suppose it can go from the package.json too? And I think the two other serverless packages aren't needed anymore either?